src/components/Home.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShuffle } from '@fortawesome/free-solid-svg-icons';
+import '../styles/Home.scss';
+
+const getRandomInt = (max) => {
+    return Math.floor(Math.random() * Math.floor(max));
+};
+const getRandomQuote = (movies) => {
```

Comment:
I don't really understand this function, it has a while loop and a very strange check.